source/backtrace/backtrace.d

```diff
@@ -135,14 +137,27 @@ private string exePath() {
   return path;
 }
 
+void printPrettyTrace(PrintOptions options = PrintOptions.init, uint framesToSkip = 1) {
+  void*[] bt = getBacktrace();
+  auto or = stdout.lockingTextWriter();
+  printPrettyTrace(bt, or, options, framesToSkip);
+}
+
 void printPrettyTrace(File output, PrintOptions options = PrintOptions.init, uint framesToSkip = 1) {
   void*[] bt = getBacktrace();
+  auto or = output.lockingTextWriter();
+  printPrettyTrace(bt, or, options, framesToSkip);
+}
+
+void printPrettyTrace(OR)(ref OR output, PrintOptions options = PrintOptions.init, uint framesToSkip = 1) {
+  void*[] bt = getBacktrace();
   printPrettyTrace(bt, output, options, framesToSkip);
 }
 
-private void printPrettyTrace(const(void*[]) bt, File output, PrintOptions options = PrintOptions.init, uint framesToSkip = 1) {
+private void printPrettyTrace(OR)(const(void*[]) bt, ref OR output, PrintOptions options = PrintOptions.init, uint framesToSkip = 1, bool insertNewlines = true) {
```

Comment:
Do you see need for `insertNewLines`? If there is, I prefer putting it in `PrintOptions`.